Userland/Applications/Screenshot/MainWindow.h

```diff
@@ -28,7 +28,9 @@ class MainWindow final : public GUI::Window {
     RefPtr<GUI::Button> m_cancel_button;
     RefPtr<GUI::Button> m_browse;
     RefPtr<GUI::RadioButton> m_selected_area;
-    RefPtr<GUI::CheckBox> m_edit_in_pixel_paint;
+    RefPtr<GUI::RadioButton> m_output_to_clipboard;
+    RefPtr<GUI::RadioButton> m_edit_in_pixel_paint;
+    RefPtr<GUI::RadioButton> m_file;
```

Comment:
Since the output items are all related, they should probably have a common prefix to make that clear. (E.g. `m_output_radio_clipboard`, `m_output_radio_pixel_paint`, `m_output_radio_file`)

Reply:
Good suggestion. Done.